core/src/main/java/org/opensearch/sql/expression/function/FunctionDSL.java

```diff
@@ -166,6 +201,55 @@ public static SerializableFunction<FunctionName, Pair<FunctionSignature, Functio
     return implWithProperties((fp, arg) -> function.apply(arg), returnType, argsType);
   }
 
+  /**
+   * Varargs Function Implementation.
```

Comment:
Please, add a small comment that this implementation considers 1..x args of the same type.